heron/common/src/python/utils/misc/serializer.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2016 Twitter. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''serializer.py: common python serializer for heron'''
+from abc import abstractmethod
+
+try:
+  import cPickle as pickle
+except:
+  import pickle
```

Comment:
Pickle is used as serializer at the tuple level or something else? pickle could be the cause of performance?